ramp-database/ramp_database/cli.py

```diff
@@ -189,6 +191,80 @@ def add_submission(config, event, team, submission, path):
                                          path)
 
 
+@main.command()
+@click.option("--config", default='config.yml', show_default=True,
+              help='Configuration file YAML format containing the database '
+              'information')
+@click.option("--config-event", required=True,
+              help="""Path to configuration file YAML format
+              containing the database information, eg config.yml""")
+@click.option('--from-disk', default=False,
+              help="""If True the event and all the related submissions
+                      will be removed from the database and from the disk;
+                      if set to False they will only be removed from
+                      the database""")
+@click.option('--force', default=False,
+              help="""If True the event will be removed from the disk
+                      (if it exists) even if it is not found
+                      in the database.
+                      This option must be used with --from-disk
+                      option to work""")
+def delete_event(config, config_event, from_disk, force):
+    """Delete event."""
+    config = read_config(config)
+
+    # check if the event_config dir exists
+    if not os.path.isfile(config_event):
```

Comment:
I am not sure that we need this. If we provide something that it is not correct it will fail, isn't it?